packages/@stylexjs/eslint-plugin/src/stylex-enforce-extension.js

```diff
@@ -107,10 +130,14 @@ const stylexEnforceExtension = {
     }
 
     return {
+      ImportDeclaration: importTracker.ImportDeclaration,
       'ExportNamedDeclaration, ExportDefaultDeclaration'(node: Node): void {
         checkExports(node);
         reportErrors(node);
       },
+      'Program:exit'() {
+        importTracker.clear();
```

Comment:
I'm not too sure this cleanup is strictly necessary, but aligned it with `packages/@stylexjs/eslint-plugin/src/stylex-sort-keys.js` which does clean up.